python/alb_ssl_policy.py

```diff
@@ -0,0 +1,389 @@
+  AWS::LoadBalancingV2::LoadBalancer
+Rule Parameters:
+  | ---------------------- | --------- | -------------------------------------------------------------------------------------- |
+  | Parameter Name         | Type      | Description                                                                            |
+  | ---------------------- | --------- | -------------------------------------------------------------------------------------- |
+  | COMPLIANT_POLICY       | Constant  | The list of ssl policy name which check whether lister configred                       |
+  | ---------------------- | --------- | -------------------------------------------------------------------------------------- |
+Feature:
+  In order to: apply ssl policy to listers
+  As: a Security Officer
+  I want: to chaging configuration of applicatio loac balancer.
+Scenarios:
+  Scenario 1:
+    Given: SslPolicy is in COMPLIANT_POLICY 
+     Then: Return COMPLIANT
+  Scenario 2:
+    Given: SslPolicy is not in COMPLIANT_POLICY
+     THEN: RETURN NON_COMPLIANT
+'''
+# Python packages
+import json
+import datetime
+import boto3
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = ['AWS::LoadBalancingV2::LoadBalancer']
+# Pre defined policy name
+COMPLIANT_POLICY = ['ELBSecurityPolicy-2016-08','ELBSecurityPolicy-FS-2018-06']
+
+#############
+# Main Code #
+#############
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+    """Form the evaluation(s) to be return to Config Rules
+
+    Return either:
+    None -- when no result needs to be displayed
+    a string -- either COMPLIANT, NON_COMPLIANT or NOT_APPLICABLE
+    a dictionary -- the evaluation dictionary, usually built by build_evaluation_from_config_item()
+    a list of dictionary -- a list of evaluation dictionary , usually built by build_evaluation()
+
+    Keyword arguments:
+    event -- the event variable given in the lambda handler
+    configuration_item -- the configurationItem dictionary in the invokingEvent
+    valid_rule_parameters -- the output of the evaluate_parameters() representing validated parameters of the Config Rule
+
+    Advanced Notes:
+    1 -- if a resource is deleted and generate a configuration change with ResourceDeleted status, the Boilerplate code will put a NOT_APPLICABLE on this resource automatically.
+    2 -- if a None or a list of dictionary is returned, the old evaluation(s) which are not returned in the new evaluation list are returned as NOT_APPLICABLE by the Boilerplate code
+    3 -- if None or an empty string, list or dict is returned, the Boilerplate code will put a "shadow" evaluation to feedback that the evaluation took place properly
+    """
+
+    ###############################
+    # Add your custom logic here. #
+    ###############################
+
+	alb_arn = configuration_item['arn']
+	alb_client = get_client('elbv2',event);

```

Comment:
Remove ; at the end

Reply:
space between comma and variable

Reply:
removed comma and add space between comma and variable.